src/notation/view/continuouspanel.cpp

```diff
@@ -200,16 +200,16 @@ void ContinuousPanel::paint(mu::draw::Painter& painter, const NotationViewContex
             newName->setTrack(e->track());
             newName->setFamily(u"FreeSans");
             newName->setSizeIsSpatiumDependent(true);
-            mu::engraving::EngravingItem::renderer()->layoutItem(newName);
+            engravingRender()->layoutItem(newName);
```

Comment:
I'm not sure if using `singleRenderer` is the best approach here. In the continuous panel, there _is_ context. And elements in the continuous panel should follow the score style. Personally I also came across this assertion failure and I fixed it locally by adding `newName->setParent` and another `setParent` somewhere. 

(And actually, that reminds us of the "explicit versus implicit parent" story, which also still needs to be cleaned up at some point.)

Reply:
Do you recommend making these items "part of the score"? (so that they have the current score as their parent) Yes, that makes sense to me.

Reply:
Yes, but of course the score itself should not be aware of these items because they are only temporary. So the items should be aware of the score, but the score not of the items.

Reply:
fixed